services/workflows-service/.eslintrc.cjs

```diff
@@ -33,6 +33,15 @@ module.exports = {
   plugins: ['@typescript-eslint'],
   rules: {
     'import/no-cycle': 'error',
+
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      {
+        argsIgnorePattern: '^_',
```

Comment:
seems like its being duplicated, can we move it to packages/config?

Reply:
great idea! pulled to root + connected via extend